modules/web/js/ballerina/diagram2/views/default/components/nodes/while-node.jsx

```diff
@@ -18,33 +18,60 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
 import CompoundStatementDecorator from './compound-statement-decorator';
-import { getComponentForNodeArray } from './../../../../diagram-util';
 import WhileNodeModel from './../../../../../model/tree/while-node';
 import DropZone from './../../../../../drag-drop/DropZone';
 import './try-node.css';
+import FragmentUtils from './../../../../../utils/fragment-utils';
+import TreeBuilder from './../../../../../model/tree-builder';
```

Comment:
use absolute paths to import modules. https://github.com/ballerinalang/composer/wiki/Developer-best-practices#use-absolute-paths-to-import-modules

Reply:
Fixed with f580f482cc82e92dd391ea225593c001ed2f5183